opcua/common/node.py

```diff
@@ -315,3 +318,9 @@ def call_method(*args, **kwargs):
         from opcua.common import methods
         return methods.call_method(*args, **kwargs)
 
+    def delete_node(self, nodeid, deleterefs=True):
+        deletenode = ua.DeleteNodesItem()
+        deletenode.NodeId = nodeid
+        deletenode.DeleteTargetReferences = deleterefs
+        results = self.server.delete_nodes([deletenode])
+        results[0].check()
```

Comment:
This is not so clear. Because there is already this function bit receives DeleteNodeItem object. Should I do checking if there is DeleteNodeItem object or NodeId? IMHO it would be nice to have some function which accepts NodeIds so it will be easier for user to use it... Maybe we can call it `delete_nodeIds` than we get following code:

`class Server:
    ...
    def delete_nodeIds(self, nodeIds, deleterefs=True):
        deletenodes = []
        for nodeId in nodeIds:
            deletenode = ua.DeleteNodesItem()
            deletenode.NodeId = nodeid
            deletenode.DeleteTargetReferences = deleterefs
            deletenodes.append(deletenode)
       
        results = self.server.delete_nodes([deletenodes])
        for result in results:
            result.check()
            

    def delete_nodes(self, deletenodes):
         #this is already implemented
`

Th